static/sw.js

```diff
@@ -0,0 +1 @@
+self.addEventListener('fetch',e=>{});
```

Comment:
Why is this needed?

Reply:
needs a service worker with a fetch event listener to be installable

Reply:
```suggestion
/* global self */

self.addEventListener('fetch', e => {})

```